setup.py

```diff
@@ -38,7 +38,7 @@ def read_version():
     return finder.version
 
 
-install_requires = ['aiohttp>=2.0']
+install_requires = ['aiohttp>=3.0']
```

Comment:
I'm not sure what will be the next version of aiohttp ?

Reply:
next version is 3.0.0, probably will be release in a month